README.md

```diff
@@ -315,6 +359,41 @@ You can access any URL or endpoint that references this address from your host.
 1. Make a change to component `X`, in its respective release (`X-release`).
 1. Run `make X-release compile images run` to build your changes and run them.
 
+#### Bumping a version in a release (or just make a change)
+
+For this example, lets suppose we want to update a release to a later tag.
+First of all checkout the desired commit:
+
+```
+host> cd src/loggregator/ && git checkout v81
```

Comment:
If you run these commands then the submodule is back to the commit is was before. We intentionally leave this in a "dirty" state because this scenario is about changing stuff in submodules during development. There is the "If everything works" section below that explains how to commit this if it's working. Do you think that is enough?